docs/docsite/rst/dev_guide/developing_module_utilities.rst

```diff
@@ -1,5 +1,3 @@
-.. _appendix_module_utilities:
```

Comment:
We probably do want an anchor here - the error this addresses was caused by an include statement, so the whole page was getting rendered twice.